src/sweeper/report.py

```diff
@@ -5,29 +5,41 @@
 A module that contains the templates for the reports and functions to format data into the report
 '''
 
-template = '''# Sweeper Empty Report
-
-**{0}** empty geometries found
-
-## ObjectId's of empty geometries
-
-`{1}`
-
-## Select features in arc
-
-`objectid in ({1})`
-'''
-
-success_template = '''# Sweeper Empty Report
-
-🚀🚀🚀 No empty geometries found 🚀🚀🚀
-
-
-if len(report.oids) > 0:
-  return string.format(template, len(report.oids), ','.join(report.oids))
-
-return success
-
-
-'sweeper-report-{}.{}.txt'.format(date.short, report_type)
-'''
+def save_report(data_dict, title, fc, path):
+    import os
+    from datetime import datetime
+    
+    now = datetime.now().strftime('%Y%m%d_%H%M%S_%f')[:-3]
+    name = f'{title}_report_{now}.txt'
+    filename = os.path.join(path, name)
+    if len(data_dict) > 0:
+        lines = []
+        with open(filename, 'w') as file:
+            file.write(f"{title.title()} Report for '{fc}' \n \n")
```

Comment:
I was going to suggest to use `os.linesep` for the `\n`'s but it looks like you they recommend not doing that... https://docs.python.org/3/library/os.html#os.linesep